### DISCUSSION TITLE
Auto3DSeg: PET-CT Configuration

### DISCUSSION THREAD

**Discussion by mibaumgartner** [Project-MONAI/MONAI#7432] [General]
Hey, great work on MONAI and Auto3DSeg. In the provided examples, `modality` (from `task.yaml`) is always set to a single value. How should that value be set for a PET-CT with 2 channels? 

Best,
Michael

**Comment by KumoLiu** [comment]
Hi @mibaumgartner, thanks for your interest here.

For muli-modalities, you can do something like this.
https://github.com/Project-MONAI/tutorials/blob/3093dec599f04657a7ae6c7a9c97eba5c910259a/nnunet/docs/input.yaml#L2
Multi-class, you can specify like this:
https://github.com/Project-MONAI/tutorials/blob/3093dec599f04657a7ae6c7a9c97eba5c910259a/auto3dseg/tasks/msd/Task04_Hippocampus/input.yaml#L10

Hope it helps, thanks.

**Comment by mibaumgartner** [comment]
Perfect, thanks!

**Comment by mibaumgartner** [comment]
Hey, replacing it with a list (as shown in the example) did not work for me in the end (it raised an error). Simply went with CT in my case ;)
